executor/aggfuncs/window_func_test.go

```diff
@@ -207,3 +182,42 @@ func (s *testSuite) TestMemRank(c *C) {
 		s.testWindowAggMemFunc(c, test)
 	}
 }
+
+func (s *testSuite) TestMemNtile(c *C) {
```

Comment:
we should moving this to `func_ntile_test.go`, note that don't forget to add the license.